### PR TITLE
Feat : Add Slider comp [Fix] Navbar and minor bugs

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

File: Components/navbar.css [hunk 1]
```diff
@@ -30,11 +24,108 @@
     font-family: inherit;
     padding: 0.5rem;
     background-color: transparent;
-    width: 25vw;
-    max-width: 600px;
+    max-width: 600px !important;
```
Comment (nareshpoonia) [inline Components/navbar.css]: You can think of avoiding important, as its not considered as a best practice

**Review by nareshpoonia — APPROVED** [review]
The Code good to merge, only one comment on important

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Hamdrive) [inline .vscode/settings.json]: This file is not needed, preferably delete it

**Review by Hamdrive — COMMENTED** [review]
Great work done, have suggested to cleanup unnecessary files
else good to go and can be merged!

File: .vscode/settings.json (hunk 2 above)
Comment (ZuberDunge) [reply .vscode/settings.json]: Thanks for noticing, Deleted it.

File: Components/navbar.css (hunk 1 above)
Comment (ZuberDunge) [reply Components/navbar.css]: Removed. Thank you.

**Pull request merged**